.github/workflows/wheel_tests_and_release.yml

```diff
@@ -177,12 +171,16 @@ jobs:
       fail-fast: false
       matrix:
         os: [windows-latest]
+        compiler: clang
         cibw_arch: ["AMD64", "x86"]
-        cibw_python: ["cp38-*", "cp39-*", "cp310-*"]
+        cibw_python: ["cp38-*", "cp39-*", "cp310-*", "cp311-*"]
         exclude:
           - os: windows-latest
             cibw_python: "cp310-*"
             cibw_arch: x86
+          - os: windows-latest
+            cibw_python: "cp311-*"
+            cibw_arch: x86
```

Comment:
```suggestion
```
I don't think we have to exclude these anymore. At the time I had skipped x86 for Python 3.10, I think NumPy was missing wheels in that case. They have since added them for all supported Python versions. Can confirm here:
https://pypi.org/project/numpy/#files

Reply:
This suggestion will likely require the `pyproject.toml` windows pinning update from #6519 in order for it to succeed